src/gengowatcher/main.py

```
@@ -111,6 +113,86 @@ def emit(self, record):
+            prompt_text = f"[label]{description}[/] [warning](required)[/]: "
+        else:
+            # Mask sensitive values
+            masked = str(current)[:4] + "..." if len(str(current)) > 8 else str(current)
+            prompt_text = f"[label]{description}[/] [{masked}]: "
+
+        console.print(prompt_text, end="")
+        new_value = input().strip()
+
+        if new_value:
+            config.set(section, option, new_value)
+            console.print(f"  [success]✓ Updated[/]")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)